src/bin/testnode.rs

```diff
@@ -51,7 +51,7 @@ fn main() {
         historian,
     )));
     eprintln!("Listening on {}", addr);
-    let threads = AccountantSkel::serve(skel, addr, exit.clone()).unwrap();
+    let threads = AccountantSkel::serve(&skel, addr, exit.clone()).unwrap();
```

Comment:
Yep, you should have rebased. Did you get the "@aeyakovenk, fyi" notification on my last PR?